services/gem/gem.tester.js

```diff
@@ -87,7 +87,7 @@ t.create('version downloads (valid, specific version)')
 
 t.create('version downloads (package not found)')
   .get('/dv/not-a-package/4.1.0.json')
-  .expectJSON({name: 'downloads@4.1.0', value: 'not found'});
+  .expectJSON({name: 'downloads', value: 'not found'});
```

Comment:
I think the easiest thing to do where a badge may have a custom label like `downloads@stable` etc is to establish the convention that the error badges always use the default label instead of trying to pass a custom label when we throw an exception. This is what I've done here. Does anyone have an example of where this would not be a good idea?

Reply:
Can't think of any badges where this would be a problem.